repair/repair.hh

```diff
@@ -46,6 +46,14 @@ public:
     repair_stopped_exception() : repair_exception("Repair stopped") { }
 };
 
+struct repair_uniq_id {
+    int id;
+    utils::UUID uuid;
+    friend std::ostream& operator<<(std::ostream& os, const repair_uniq_id& x) {
```

Comment:
Please move this to .cc.

Reply:
You can add `std::strong_ordering operator<=>(repair_uniq_id) const = default;` and get automatic compares.

Reply:
OK.

Reply:
Although I love spaceship a lot, here I wanted a print operator not less operator.